packages/react-core/src/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -205,21 +207,21 @@ Variant styling can be applied to split button toggles to adjust their appearanc
 
 ```
 
-### Split button toggle with checkbox label
+### Split button toggle with checkbox and text label
 
-To display text in a split button menu toggle, add a label to the `items` property of `splitButtonOptions`.
+To add a text label to a split button toggle, pass `children` to the `<MenuToggle>` component.
 
-```ts file='MenuToggleSplitButtonCheckboxWithText.tsx'
+```ts file='MenuToggleSplitButtonCheckboxWithToggleText.tsx'
 
 ```
 
-### Split button toggle with checkbox and toggle button text
+### Split button toggle with checkbox and clickable text label  
```

Comment:
I really like "a labelled checkbox"! 

For the "clickable" example - I was trying to find a way to make things more concise/short and avoid writing "toggle" multiple times in the heading, but definitely want to be precise
- Is "split button toggle with checkbox label that toggles" accurate? If not, I can go with Eric's suggestion instead!

Reply:
Yeah having to repeat the "Split button toggle" text makes things difficult here 😆 Could "checkbox label that toggles" imply that the text is toggling the checkbox?

Do you think updating the examples to "Split toggle..." rather than "Split button toggle..." would help at all? I feel like ideally it'd be great to have a separate examples section for "Split button toggle examples" then the example titles could just omit the "Split button toggle" part, but I feel like there's still the issue with how our examples are rendered in Org that would cause some issue with that.

Reply:
"a separate examples section for "Split button toggle examples"" is exactly what I was wishing we could do 😭 but yeah I remember there being org restrictions around that

I like the idea of going with "split toggle". It's a small change, but it'll help cut down on wordiness, while still getting the point across!

I went with your previous selection "Split toggle with checkbox and toggle text label" & updated the other example headings in my last commit. I think I'm still having my own comprehension issues with these examples that get in the way of my suggestions lol - so as long as what we have makes sense from a dev standpoint I'm happy!

Reply:
To be fair the examples are *very* similar, so I definitely get any confusion or anything with them! Thanks for making these updates 🪨 ⭐